lib/PuppeteerSharp/BrowserData/InstalledBrowser.cs

```diff
@@ -62,7 +53,16 @@ public string GetExecutablePath()
             var installationDir = Cache.GetInstallationDir(Browser, Platform, BuildId);
             return Path.Combine(
                 installationDir,
-                _executablePathByBrowser[Browser](Platform, BuildId));
+                GetExecutablePath(Browser)(Platform, BuildId));
         }
+
+        private static Func<Platform, string, string> GetExecutablePath(SupportedBrowser browser) => browser switch
```

Comment:
We could also add the `platform` and `buildId` parameters and avoid the intermediate `Func<Platform, string, string>` which also makes the method signature prettier.